account/client.go

```diff
@@ -51,18 +39,9 @@ func GetByID(id string, params *stripe.AccountParams) (*stripe.Account, error) {
 }
 
 func (c Client) GetByID(id string, params *stripe.AccountParams) (*stripe.Account, error) {
-	var body *form.Values
-	var commonParams *stripe.Params
-
-	if params != nil {
-		commonParams = &params.Params
-		body = &form.Values{}
-		form.AppendTo(body, params)
-	}
-
+	path := stripe.FormatURLPath("/accounts/%s", id)
 	account := &stripe.Account{}
-	err := c.B.Call("GET", stripe.FormatURLPath("/accounts/%s", id), c.Key, body, commonParams, account)
-
+	err := c.B.Call("GET", path, c.Key, params, account)
```

Comment:
Yeah. Every client package defines its own `Client` which is where a `Key` is configured:

``` go
type Client struct {
	B   stripe.Backend
	Key string
}
```

`Client` has a reference to a `Backend`, but not vice versa, so that backend can't reach back into the client to get at the `Key` field. That's why it's injected with every call.

I don't think I would have designed it this way, and there might be something we can do to refactor to remove even more boilerplate, but I don't want to mess around with this too much for now.